dev-docs/RFCs/v7.x/spring-based-transitions-rfc.md

```diff
@@ -69,13 +98,44 @@ new Layer({
 });
 ```
 
-[SAY SOME STUFF HERE ABOUT `onStart`, `onEnd`, AND `onInterrupt` CALLBACKS]
-
 Note the following change to the `enter` callback:
 
 The `enter` callback receives _one_ argument. A `toValue` (TypedArray) which represents the new value to transition to, for the current vertex. The `fromChunk` value that is supplied to `enter` for interpolation-based transitions is not supplied for spring-based transitions because no "from value" is maintained for the transition. 
 
 
 ## Implementation
 
-[FILL ME IN]
+A minimal spring-based transition system looks something like this:
+
+```js
+function createSpring (stiffness, damping, initialValue) {
```

Comment:
In deck.gl we usually aim for declarative APIs `{type: 'spring', stiffness: 1, dampening: 0.8, initialValue}`

Reply:
Gotcha! This sample code is really just to give a sense of the frame-to-frame animation logic. I'm not suggesting we expose this API to Deck users. The proposed API is up in the `Approach` section. I should have been clearer about that!